src/EA.Iws.Database/scripts/Update/0005-Beta/Sprint3/20190108-141500-add-lookup-audit-data.sql

```diff
@@ -33,13 +33,17 @@ VALUES
 (21,'Customs office'),
 (22,'Amounts and dates'),
 (23,'Chemical composition'),
-(24,'Process of generation'),
-(25,'Physical characteristics'),
-(26,'Basel/OECD code'),
-(27,'EWC codes'),
-(28,'Y codes'),
-(29,'H or HP codes'),
-(30,'UN classes'),
-(31,'UN numbers'),
-(32,'Other codes'),
-(33,'Waste recovery')
\ No newline at end of file
+(24,'Chemical composition - Continued'),
```

Comment:
Could you please add these new changes in a new script file? This particular script would have been executed already so any further releases will no longer trigger this script. I'd suggest to leave this existing one as it is then in your new script you'd be running an `UPDATE` on Ids from 24-33 then running an `INSERT` from 34-37 - if that makes sense?

Reply:
Oh, yh of course!